e2etest/bgptests/bgp.go

```diff
@@ -602,19 +439,20 @@ var _ = ginkgo.Describe("BGP", func() {
 						Addresses: poolAddresses,
 					},
 				},
-				Peers:       withBFD(peersForContainers(frrContainers, pairingFamily), bfd.Name),
+				Peers:       metallb.WithBFD(metallb.PeersForContainers(FRRContainers, pairingFamily), bfd.Name),
```

Comment:
WDYT about moving the bfd tests to a separate file (still under bgptests)?

Reply:
Sounds good

Reply:
Uhm, that would require duplicating the describe part and its beforeEach / AfterEach, and all the local variables, which is not ideal.